content/en/dashboards/functions/timeshift.md

```diff
@@ -22,6 +22,24 @@ For example, if you wanted to use this to compare current system load with load
 timeshift(avg:system.load.1{*}, -1209600)
 ```
 
+## Calendar shift
+
+<div class="alert alert-info">The calendar shift feature is now available for all data sources.</div>
```

Comment:
```suggestion
<div class="alert alert-info">The calendar shift feature is available for all data sources.</div>
```

Reply:
The idea behind this banner is to highlight that this support is new, because earlier this was supported only for some data sources. We would be removing the banner after a while.
if you think "now" shouldn't be here, I'd prefer not having a banner at all, as it should be assumed that everything works for every data source, unless stated differently.

Reply:
Ah then yes, I think in this case we should remove this banner.